gcp-deployer/machine_setup_configs.yaml

```diff
@@ -179,10 +23,23 @@ items:
       apt-get install -y \
           socat \
           ebtables \
-          docker.io \
           apt-transport-https \
           cloud-utils \
           prips
+
+      function install_configure_docker () {
```

Comment:
Wouldn't similar changes be required for the nodes?

Reply:
I added the change in for nodes.